tests/test_policy_gradient.py

```diff
@@ -28,10 +28,10 @@ def test_pg_fully_observable_tmaze():
         v_0 = agent.policy_improvement(amdp, lr)
 
     learnt_pi = softmax(agent.pi_params, axis=-1)
-    assert np.allclose(learnt_pi[:-3, 2], np.ones_like(learnt_pi[:-3, 2]), atol=1e-2), \
+    assert np.allclose(learnt_pi[:-3, 2], np.ones_like(learnt_pi[:-3, 2]), atol=5e-2), \
```

Comment:
0.05 is a pretty big range. Are these asserts still useful?

Reply:
Yes they are! When we were at `atol=1e-2`, there were certain seeds where this assertion would fail, but just barely.